packages/ui/src/tailwind/tailwind-plugin.ts

```diff
@@ -129,11 +129,11 @@ export const WestpacUIKitBasePlugin = plugin.withOptions(
             switch: '0.1875rem 0 0.375rem 0 rgba(0,0,0,0.53)',
           },
           maxWidth: {
-            xsl: BREAKPOINTS.xsl,
-            sm: BREAKPOINTS.sm,
-            md: BREAKPOINTS.md,
-            lg: BREAKPOINTS.lg,
-            container: '1320px', //1200 (lg) + 60 (paddingHorizontal) + 60 (paddingHorizontal)
+            xsl: '47.25rem',
+            sm: '60.75rem',
```

Comment:
Why we can't use the pre-defined const?

Reply:
because the breakpoint and container size are two different things.

if the screen is 1440px the container has to be a little less than that.